content/blog/2016/2016-05-26-pipeline-snippetizer.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: New display of Pipeline’s “snippet generator”
```

Comment:
It would appear that whatever editor you're using for asciidoc is throwing smart quotes in, blergh.